tools/clang/test/CodeGenDXIL/hlsl/linalg/make-interp-vec-errors.hlsl

```diff
@@ -0,0 +1,38 @@
+// RUN: not %dxc -T lib_6_9 %s 2>&1 | FileCheck %s
+
+#include "linalg.h"
```

Comment:
```suggestion
#include <linalg.h>
```

Reply:
I've just removed the line entirely since it's unneeded, the file is included by default when compiling a shader.

Reply:
Oh wow, so just putting in that directory makes it get included by default?  That's awesome!